src/redis-cli.c

```diff
@@ -3358,16 +3358,17 @@ void cliLoadPreferences(void) {
 }
 
 /* Some commands can include sensitive information and shouldn't be put in the
- * history file. Currently these commands are include:
+ * history file. Currently these commands are:
```

Comment:
revert the changes to this file when changing to use AUTH

Reply:
Reverted